lms/static/styles/moodle_pages/moodle_pages.scss

```diff
@@ -0,0 +1,242 @@
+// This stylesheet contains a re-implementation of some of Moodle styles for
+// pages, so that Moodle pages served through our LMS app + Via look close to
+// how the page looks in Moodle.
+//
+// The markup generated for pages does not include specific classes, so we only
+// need to define styles for HTML elements.
+//
+// The styles defined here are mainly extracted from the next Moodle files:
+// * Base definition:
+//   * https://github.com/moodle/moodle/blob/main/theme/boost/scss/bootstrap/_reboot.scss
+// * Variables:
+//   * https://github.com/moodle/moodle/blob/main/theme/boost/scss/preset/default.scss
+//   * https://github.com/moodle/moodle/blob/main/theme/boost/scss/bootstrap/_variables.scss
+
+body {
+  // Defined as $font-family-sans-serif
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
+    'Helvetica Neue', Arial, 'Noto Sans', 'Liberation Sans', sans-serif,
+    'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
+  font-size: 0.9375rem; // Defined as `$font-size-base`
+  line-height: 1.5;
+  color: #1d2125; // Defined as `$gray-900`
```

Comment:
This is a SASS file, so you might as well define matching variables rather than write out comments to the same effect. I see that `$font-size-base` really is defined as `0.9375rem` upstream. Assuming a base size of 16px, that equals 15px.